flink-yarn/src/main/scala/org/apache/flink/yarn/YarnJobManager.scala

```diff
@@ -34,7 +34,6 @@ import org.apache.flink.runtime.instance.InstanceManager
 import org.apache.flink.runtime.jobmanager.scheduler.{Scheduler => FlinkScheduler}
 import org.apache.flink.runtime.jobmanager.{JobManager, SubmittedJobGraphStore}
 import org.apache.flink.runtime.leaderelection.LeaderElectionService
-import org.apache.flink.runtime.metrics.MetricRegistryImpl
```

Comment:
Is this irrelevant? Would prefer this as a separate hotfix.

Reply:
It already separate a hotfix. We have two commits here.